src/ResourceManager/Compute/Commands.Compute/Models/PSVirtualMachine.cs

```diff
@@ -88,34 +91,7 @@ public string ResourceGroupName
         // Gets or sets the storage profile.
         public StorageProfile StorageProfile { get; set; }
 
-        [JsonIgnore]
```

Comment:
There are numerous cmdlets that output an object of type `PSVirtualMachine`, and existing scripts that use this object will no longer be able to use these properties, which means it's a breaking change.